r2/r2/models/builder.py

```diff
@@ -943,7 +943,7 @@ def update_candidates(self, candidates, sorter, to_add=None):
     "qa": operators.desc('_qa'),
     "hot": operators.desc('_hot'),
     "top": operators.desc('_score'),
-    "random": operators.shuffled('_confidence'),
+    "random": operators.shuffled('_random'),
```

Comment:
I think this dict is only used to figure out which precomputed comment order to use. The one you want to change is in `SortMenu` https://github.com/reddit/reddit/blob/master/r2/r2/lib/menus.py#L560